tests/test_create.py

```diff
@@ -932,7 +932,7 @@ def test_bad_anaconda_token_infinite_loop(self):
         # First, confirm we get a 401 UNAUTHORIZED response from anaconda.org
```

Comment:
Does this comment need an update too? :stuck_out_tongue:

Reply:
Yup! Thanks for keeping me honest @nehaljwani :)